app/src/main/java/lt/vilnius/tvarkau/fragments/NewReportFragment.kt

```diff
@@ -541,9 +546,9 @@ class NewReportFragment : BaseFragment(),
                 setProgressStyle(ProgressDialog.STYLE_SPINNER)
                 setCancelable(false)
             }
-        } else {
-            progressDialog?.show()
         }
+
+        progressDialog?.show()
```

Comment:
KT can't now it this property wasn't modified in another thread.
So it's either `!!` or `?`